testing/topgit/APKBUILD

```diff
@@ -0,0 +1,27 @@
+# Contributor: Joseph Benden <joe@benden.us>
+# Maintainer: Joseph Benden <joe@benden.us>
+
+pkgname=topgit
+pkgver=0.19.12
+pkgrel=0
+pkgdesc="A different patch queue manager"
+arch="noarch"
+url="https://github.com/mackyle/topgit"
+license="GPL"
+options="!check !strip"
+depends="git gawk sed"
+makedepends="gawk sed perl"
```

Comment:
gawk and sed are in depends= no need to be in makedepends

Reply:
Ok, thanks!